packages/turbo-repository/rust/src/lib.rs

```diff
@@ -64,14 +61,14 @@ impl PackageManagerRoot {
         // match rather than map/map_err due to only the Ok variant implementing "Copy"
         // match lets us handle each case independently, rather than forcing the whole
         // value to a reference or concrete value
-        match self.repo_state.package_manager.as_ref() {
+        match self.workspace_state.package_manager.as_ref() {
             Ok(pm) => Ok((*pm).into()),
             Err(e) => Err(napi::Error::from_reason(format!("{}", e))),
         }
     }
 
     #[napi]
-    pub async fn packages(&self) -> std::result::Result<Vec<Package>, napi::Error> {
+    pub async fn find_packages(&self) -> std::result::Result<Vec<Package>, napi::Error> {
```

Comment:
Is the thought behind adding the `find` that it better indicates that this function does work and isn't just a getter?

Reply:
Is this just a getter? My understanding was that this async function was calling `package_manager.get_package_jsons()` underneath, which seems to traverse?